lib/internal/per_context/domexception.js

```diff
@@ -10,12 +10,27 @@ const {
   TypeError,
 } = primordials;
 
-class ERR_INVALID_THIS extends TypeError {
-  constructor(type) {
-    super('Value of "this" must be of ' + type);
-  }
-
-  get code() { return 'ERR_INVALID_THIS'; }
+function throwInvalidThisError(Base, type) {
```

Comment:
Is the change to domexception.js related to the urlsearchparams change? If it is, it's not clear how. Also, it would likely be better to separate this out into a separate commit in the PR if it is related.

Reply:
This modification is to pass `urlsearchparams-contructor.any.js` too.

Reply:
`URLSearchParams` constructor should throw exactly `TypeError` if any Error occurrs but not inheritance.

Reply:
https://github.com/nodejs/node/blob/master/test/fixtures/wpt/url/urlsearchparams-constructor.any.js#L25-L26